hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/sigs/utils/MiscCryptoUtils.java

```diff
@@ -18,20 +18,45 @@
 
 import static com.hedera.node.app.service.mono.utils.EntityIdUtils.EVM_ADDRESS_SIZE;
 
+import com.sun.jna.ptr.LongByReference;
+import java.nio.ByteBuffer;
 import java.util.Arrays;
 import org.bouncycastle.jcajce.provider.digest.Keccak;
-import org.bouncycastle.jce.ECNamedCurveTable;
-import org.bouncycastle.jce.spec.ECNamedCurveParameterSpec;
-import org.bouncycastle.math.ec.ECCurve;
+import org.hyperledger.besu.nativelib.secp256k1.LibSecp256k1;
 
 public class MiscCryptoUtils {
+
+    /**
+     * Record for caching thread local variables to avoid allocating memory for each verification.
+     */
+    private record ThreadLocalCache(
+            LibSecp256k1.secp256k1_pubkey pubKey,
+            byte[] uncompressedPublicKeyInput,
+            ByteBuffer uncompressedPublicKeyByteBuffer,
+            LongByReference length) {
+        public ThreadLocalCache() {
+            this(new LibSecp256k1.secp256k1_pubkey(), new byte[65], ByteBuffer.allocate(65), new LongByReference());
```

Comment:
Should use the named constant below instead of 65.

Reply:
Fixed